cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/co/cask/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -43,6 +43,7 @@
   private final List<ETLStage> postActions;
   // for backwards compatibility
   private final List<ETLStage> actions;
+  private final boolean isService;
```

Comment:
nit: rename it to `service`. I think we don't call boolean variables starting with `is` which is used for methods.